lms/djangoapps/certificates/tasks.py

```diff
@@ -41,16 +41,22 @@ def generate_certificate(self, **kwargs):
     course_key = CourseKey.from_string(kwargs.pop('course_key'))
     expected_verification_status = kwargs.pop('expected_verification_status', None)
     allowlist_certificate = kwargs.pop('allowlist_certificate', False)
+    v2_certificate = kwargs.pop('v2_certificate', False)
```

Comment:
Do you want to add this to the list above (like `allowlist_certificate`, ~36), or not bother since this is 🤞 hopefully temporary?

Reply:
I can update the doc if that would help. This is just the first pass (to break the work into smaller PRs) and the param doesn't do anything yet.